rudder-core/src/main/scala/com/normation/rudder/domain/reports/ExpectedReports.scala

```diff
@@ -44,6 +44,22 @@ import com.normation.inventory.domain.NodeId
 
 
 /**
+ * Remark on above: RuleExpectedReport does not
+ * map to anything usefull today. We would like to
+ * have something like:
+ */
+case class RuleNodeExpectedReports(
```

Comment:
I am not very fan of the name as it has no links with Node, and I think it may be hard to maintain with time. (very confusing having two class with almost the same name)

Is it possible to remove "RuleExpectedReports" and rename it or is it still used somehow ?